lib/app_store_connect/requests/v1/in_app_purchase_review_screenshot/update.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module AppStoreConnect
+  module Requests
+    module V1
+      module InAppPurchaseReviewScreenshot
+        class Update < CreateRequest
```

Comment:
You'll need to inherit from `Request:Body` now. `CreateRequest` and `UpdateRequest` were consolidated in #146

Reply:
Addressed in 8220aeb